static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.ts

```diff
@@ -140,17 +140,18 @@ export const init = (): Promise<void> => {
 					{ async: false },
 				);
 			}
+			return Promise.resolve();
 		});
-		return Promise.resolve();
 	};
 
 	if (commercialFeatures.dfpAdvertising) {
-		// A promise error here, from a failed module load,
-		// could be a network problem or an intercepted request.
-		// Abandon the init sequence.
-		setupAdvertising().then(adFreeSlotRemove).catch(removeSlots);
-
-		return Promise.resolve();
+		return (
+			setupAdvertising()
+				// on success, remove slots for ad-free users
+				.then(adFreeSlotRemove)
+				// on error, remove all slots
+				.catch(removeSlots)
+		);
```

Comment:
Why is this wrapped in parentheses?

Is `return (Promise.resolve())` the same as `return Promise.resolve()`?

Reply:
Prettier inserts the parens because the promise chain is on multiple lines but I can't seem to find the relevant rule in the prettier documentation.
Good question - I was blindly trusting prettier but it looks as though `return (x)` is indeed the same as `return x` in js. I think the parens are functioning as a [grouping operators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Grouping) in this case and since there is only one expression, the grouping has not effect.

Reply:
Fine, that’s what I thought as well! I always get a little bit confused by grouping operators. Again, pretty sage when there’s only one expression.